src/middlewares/errors.js

```diff
@@ -0,0 +1,6 @@
+import logger from "../utils/logger.js";
+
+export default function errorsMiddleware(err, req, res, next) {
+  logger.error(err.stack);
+  res.status(500).send('Something went wrong!');
```

Comment:
Don't we want to send some content of `err`, such as `err.message`? 🤔 
```suggestion
  res.status(500).json({ error: err.message });
```